app/views/govuk_component/docs/task_list.yml

```diff
@@ -84,7 +84,7 @@ examples:
     description: Sections have a H2 by default, but this can be changed. The heading level does not change any styling.
```

Comment:
Should this read `Steps have a H2 by default...`?

Reply:
Good spot, thanks.